sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -94,24 +94,18 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     return await this.save(institution);
   }
 
-  async updateInstitution(userInfo: UserInfo, institutionDto: InstitutionDto) {
-    const account = await this.bceidService.getAccountDetails(
-      userInfo.idp_user_name,
-    );
-
-    if (account == null) {
-      // Due to any connection or any other integration error retrieval of account info failed.
-      this.logger.error(
-        "Account information could not be retrieved from BCeID",
-      );
-      throw new InternalServerErrorException(
-        `Unable to fetch account information from BCeID`,
-      );
-    }
+  async getInstitute(userInfo: { userName: string }): Promise<Institution> {
```

Comment:
Can this method be named as something like getInstitutionByUserName and have the parameter as userName only?
`async getInstitutionByUserName(userName: string): Promise<Institution>`

Reply:
DONE